criu/pie/restorer.c

```diff
@@ -782,12 +770,22 @@ __visible long __export_restore_thread(struct thread_restore_args *args)
 	rt_sigframe = (void *)&args->mz->rt_sigframe;
 
 	if (args->cg_set != -1) {
+		int err = 0;
```

Comment:
nit: I prefer a new empty line between the variable declaration and following statements.